browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -470,7 +470,7 @@ IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
 }
 
 IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
-                       DISABLED_NavigationCookiesArePartitioned) {
+                       NavigationCookiesArePartitioned) {
```

Comment:
I think we should change the name of this test because "navigation cookies" doesn't mean anything to me. This is presumably for network cookies?

Reply:
actually what we really need is to check network and JS for all tests now

Reply:
extended/added tests to check network and JS cookie set.